src/DetailsView/components/settings-panel/settings/issue-filing/github-bug-settings-ux.tsx

```diff
@@ -8,10 +8,10 @@ import { SettingsProps } from '../settings-props';
 
 export const GitHubBugSettingsUx = NamedSFC<SettingsProps>('GitHubBugSettingsUx', props => {
     const onGitHubRepositoryChange = (event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>, newValue?: string) => {
-        props.deps.userConfigMessageCreator.setBugServiceProperty('gitHub', 'repository', newValue);
+        props.deps.userConfigMessageCreator.setIssueServiceProperty('gitHub', 'repository', newValue);
     };
 
-    const getBugServiceProperty = (bugService: string, propertyName: string) => {
+    const getIssueServiceProperty = (bugService: string, propertyName: string) => {
```

Comment:
Can we name it `getIssueFillingServiceProperty` rather than `getIssueServiceProperty`; since the service is more for Issue filling and not only about issue. thoughts?